test/scheduler/onlinetesting/EvaluatePhase1ResultJobSpec.scala

```diff
@@ -36,15 +36,15 @@ class EvaluatePhase1ResultJobSpec extends PlaySpec with MockitoSugar with ScalaF
 
   "Scheduler execution" should {
     "evaluate applications ready for evaluation" in new TestFixture {
-      when(mockEvaluateService.nextCandidatesReadyForEvaluation(any[Int])).thenReturn(Future.successful(Some(apps.toList, passmark)))
+      when(mockEvaluateService.nextCandidatesReadyForEvaluation(any[Int])).thenReturn(Future.successful(Some((apps.toList, passmark))))
```

Comment:
@Lansalo Yes. The signature is ```Future[Option[(List[ApplicationPhase1ReadyForEvaluation],Phase1PassMarkSettings)]]```,
 i.e a tuple. Not putting a tuple in compiles, because the compiler infers a tuple, but it generates a warning, and should not be done